metricbeat/module/memcached/stats/_meta/fields.yml

```diff
@@ -53,12 +53,13 @@
       description: >
         Number of "set" commands serviced since startup.
 
+    # ECS?
```

Comment:
Pretty sure these are monotonic counters. Nothing like this at this time in ECS. I'd say leave as is

Reply:
+1